### ISSUE TITLE
conf

### ISSUE THREAD

**Issue by microchila** [ultralytics/ultralytics#15979]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

How do you get the classification confidence and object confidence of a box, the output conf attribute is currently the product of the final two, right?

**Comment by Y-T-G** [comment]
No. There's no objectness score in YOLOv8. Just class confidence. YOLOv5 had objectness score. YOLOv8 doesn't.

**Comment by microchila** [comment]
Isn't the final detection confidence of YOLO series calculated by multiplying object confidence and classification confidence? I see that there is an attribute prob which is the classification confidence. @Y-T-G

**Comment by Y-T-G** [comment]
Not for YOLOv8. YOLOv8 just uses classification confidence.
